src/acceptance-test/java/org/zalando/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -36,30 +40,54 @@ public SubscriptionCursor getCursor() {
         return unmodifiableList(events);
     }
 
+    public StreamMetadata getMetadata() {
```

Comment:
If it can be null - the getter should be marked as `@Nullable` or it can be Optional.